coderd/healthcheck/provisioner.go

```diff
@@ -26,7 +27,13 @@ type ProvisionerDaemonsReport struct {
 	Dismissed bool             `json:"dismissed"`
 	Error     *string          `json:"error"`
 
-	ProvisionerDaemons []codersdk.ProvisionerDaemon `json:"provisioner_daemons"`
+	Items []ProvisionerDaemonsReportItem `json:"items"`
```

Comment:
In theory, this is API breaking, right?

Reply:
Yeah in theory, but the modifications were only merged this morning :-)